buddyboss/buddyboss-platform
# PROD-866#2041 - Fix standalone subforums(with no groups) are still accessible even if it is under a parent forum that is under a private group

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2041.

### How to test the changes in this Pull Request:

1. Check the architecture forum for example and unlink it to a group
2. Set the architecture's parent forum as nature lovers which is a forum under a private group
3. View as the user jessica that is not an admin
4. Notice that the Forum architecture is not vis8ible in the forum list
5. However, when accessing it via its link is possible
https://www.loom.com/share/c706c7595b734741ba205438d0029af0

### Proof Screenshots or Video

https://www.loom.com/share/c706c7595b734741ba205438d0029af0

### Other information:




### Changelog entry

> Fixed standalone subforums(with no groups) are still accessible even if it is under a parent forum that is under a private group
